### PR TITLE
Semantic ui buttons revamp

### PR DESCRIPTION
**Pull request by fraincs**
Issue:

Buttons styles were all over the place.

## What I did

Revamp all the css from the semantic ui theme button. Added way more Chromatic tests.

Fixed
https://github.com/gsoft-inc/sg-orbit/issues/127
https://github.com/gsoft-inc/sg-orbit/issues/103
https://github.com/gsoft-inc/sg-orbit/issues/102
https://github.com/gsoft-inc/sg-orbit/issues/68

### CODE REVIEW

**Comment by patricklafrance** [comment]
Looks great!

Could you please add the matching issues in the PR description?

**Comment by patricklafrance** [comment]
The following cases are broke in the docs:

image

**Comment by patricklafrance** [comment]
Also, please remove the dimension classes on the Icons in the docs examples since it's not required anymore.

**Comment by patricklafrance** [comment]
The naked chromatic snapshot should includes a few "custom colors" override to make the styles can be overrided

**Comment by patricklafrance** [comment]
For groups example in the docs, I understand that the groups are not 100% ready but look at the icon with a tag, it the margins are not good. From what I understand a button with a label should work at this point?

**Comment by fraincs** [comment]
> For the group example in the docs, I understand that the groups are not 100% ready but look at the icon with a tag, the margins are not good. From what I understand a button with a label should work at this point?
> 
> You should also add a few chromatic tests for this.

Would you suggest a separate test?

**Comment by fraincs** [comment]
> The naked chromatic snapshot should includes a few "custom colors" override to make the styles can be overrided

I would create another shot, in order not to add unnecessary complexity in the tests.

File: packages/react-components/components/button/tests/chromatic/buttons.jsx [hunk 1]
```diff
@@ -0,0 +1,138 @@
+import { Button } from "@orbit-ui/react-button/src";
+import { CalendarIcon, CalendarIcon24, SignoutIcon24 } from "@orbit-ui/icons";
+import { Label } from "@orbit-ui/react-label/src";
+
+export const ThemeButtons = props => {
```
Comment (patricklafrance) [inline packages/react-components/components/button/tests/chromatic/buttons.jsx]: Rename to Buttons

File: packages/react-components/components/button/tests/chromatic/buttons.jsx [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    return (
+        <>
```
Comment (patricklafrance) [inline packages/react-components/components/button/tests/chromatic/buttons.jsx]: Fragment is not required here

**Pull request merged**